api-js/src/affiliation/mutations/__tests__/update-user-role.test.js

```diff
@@ -688,7 +684,9 @@ describe('update a users role', () => {
           )
 
           const error = [
-            new GraphQLError('Unable to update users role. Please try again.'),
+            new GraphQLError(
+              'Unable to update users role, in unknown organization.',
```

Comment:
See above!

Reply:
Unable to update role: organization unknown